amazon-sqs/5.11/modules/ROOT/pages/amazon-sqs-connector-reference.adoc

```diff
@@ -1174,107 +1282,120 @@ Defines a configuration for TLS, which can be used from both the client and serv
+| Id a| String | Identifier for this particular receipt handle. This is used to communicate the result. The IDs of a batch request must be unique within the request.  |  |
+| Receipt Handle a| String | Receipt handle for this entry. |  |
+| Visibility Timeout a| Number | New value (in seconds) for the message's visibility timeout. |  |
 |===
 
 [[DeleteMessageBatchRequestEntry]]
 === Delete Message Batch Request Entry
 
+Used to delete messages in batch.
+
 [%header,cols="20s,25a,30a,15a,10a"]
 |===
 | Field | Type | Description | Default Value | Required
-| Id a| String | An identifier for this particular receipt handle. This is used to communicate the result. Note that the **Id**s of a batch request need to be unique within the
- request.  |  |
-| Receipt Handle a| String | The receipt handle for this entry. |  |
+| Id a| String | Identifier for this particular receipt handle. This is used to communicate the result. The IDs of a batch request must be unique within the request.  |  |
+| Receipt Handle a| String | Receipt handle for this entry. |  |
 |===
 
 [[Message]]
 === Message
 
-The output attributes for a sent message.
+Output attributes for a sent message.
 
 [%header,cols="20s,25a,30a,15a,10a"]
 |===
 | Field | Type | Description | Default Value | Required
-| body a| String | The message body to send |  |
-| deduplicationId a| String | The token for deduplicating sent messages. If multiple messages are sent with the same deduplication ID, Amazon SQS accepts the messages successfully, but delivers only the first message during the 5-minute deduplication interval.   |  |
-| delaySeconds a| Number |  Number of seconds that a specific message is delayed. Valid values are 0 through 900.
- If no value is specified, the default value for the queue is used. This parameter can
- only be set on a queue level--when FifoQueue is set, DelaySeconds cannot be set per message. |  |
-| groupId a| String | Group this message belongs to in the FIFO Queue. This parameter applies only to FIFO queues.
- When creating a FIFO queue this parameter is required.|  |
-| id a| String | A unique identifier for the message |  |
-| message Attributes a| Object | A map of typed key-value pairs to send as message attributes. You must specify a value, key, and data type for each entry. |  |
-| receipt Handle a| String | The receipt handle for this entry. |  |
+| body a| String | Message body to send. |  |
+| deduplicationId a| String | Token for deduplicating sent messages. If multiple messages are sent with the same deduplication ID, Amazon SQS accepts the messages successfully, but delivers only the first message during the 5-minute deduplication interval.   |  |
+| delaySeconds a| Number |  Number of seconds that a specific message is delayed. The valid values are 0 through 900.
+ If no value is specified, the default value for the queue is used. This parameter can be set only on a queue level-- when FifoQueue is set, DelaySeconds cannot be set per message. |  |
```

Comment:
Maybe split this text into two sentences since the sentence parts don't seem that related.